### PR TITLE
[MOBILE-1819] Update native SDKs and release 8.1.0

### PR DESCRIPTION
**Pull request by BrianBatchelder**
### What do these changes do?
Update native SDKs to latest versions and create release.

### Why are these changes necessary?
Some new Flutter features to roll out, plus a few native SDK fixes.

### How did you verify these changes?
Local CI. Received push on both iOS and Android.

### Anything else a reviewer should know?
- I fixed a small error in the DEV_README
- I had to make changes to the iOS example project to get it to run and receive a push
- I made a fix to the docs build that will hopefully publish the docs as we expect them.

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,5 +1,15 @@
 # Flutter Plugin Changelog
 
+## Version 3.1.0 - August 18, 2020
+Patch release adding date attributes, custom event properties, named user attributes, support for the new Android plugins APIs and bundling the following SDK updates:
```
Comment (marc-scig) [inline CHANGELOG.md:4]: Minor release?

File: example/ios/Runner.xcodeproj/project.pbxproj [hunk 2]
```diff
@@ -389,7 +390,7 @@
 					"$(inherited)",
 					"$(PROJECT_DIR)/Flutter",
 				);
-				PRODUCT_BUNDLE_IDENTIFIER = com.airship.airshipExample;
+				PRODUCT_BUNDLE_IDENTIFIER = com.urbanairship.richpush;
```
Comment (marc-scig) [inline example/ios/Runner.xcodeproj/project.pbxproj:393]: do we actually want this bundle ID in the wild?

Comment (ralepinski) [reply example/ios/Runner.xcodeproj/project.pbxproj:393]: it matches our other samples

Comment (BrianBatchelder) [reply example/ios/Runner.xcodeproj/project.pbxproj:393]: https://github.com/urbanairship/ios-library/blob/2c70f023772d06ab87a8cfd80e67f0931bfcb300/SwiftSample/SwiftSample.xcodeproj/project.pbxproj#L1286

**Pull request merged**